### ISSUE TITLE
include one copyright message per bundle

### ISSUE THREAD

**Issue by fantapop** [okta/okta-auth-js#828]
I've been trying to figure out how I can reduce my bundle size when included okta-auth-js on my frontend.  I noticed that bundles include large copyright notices for each file.  The `dist/okta-auth-js.umd.js` file includes 40KB of these notices.  I don't know if this is possible from a legal perspective but it would be great to strip the Copyright notices during the build and then add a single notice at the top of the bundle.

```
 grep -c Copyright node_modules/@okta/okta-auth-js/dist/okta-auth-js.umd.js
69
```

**Comment by shuowu** [comment]
@fantapop Thanks for catching the issue! We definitely should optimize the bundle size to keep one copyright banner per bundle.

Internal Ref: OKTA-405342

**Comment by shuowu** [comment]
Resolved in version 5.2.0

https://github.com/okta/okta-auth-js/blob/master/CHANGELOG.md#520
